app/tasks/sdks.php

```diff
@@ -225,8 +225,8 @@
                 \exec('mkdir -p '.$resultExamples.' && cp -r '.$result.'/docs/examples '.$resultExamples);
                 Console::success("Copied code examples for {$language['name']} SDK to: {$resultExamples}");
 
-                \exec('rm -rf '.$result);
-                Console::success("Removed source code directory '{$result}' for {$language['name']} SDK");
+                // \exec('rm -rf '.$result);
+                // Console::success("Removed source code directory '{$result}' for {$language['name']} SDK");
```

Comment:
Since its in GitIgnore now - we can delete it before generating 👍🏻

Reply:
Got it :+1: